Xcode/check-xcode-version.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# Compares the currently used Xcode version to build the project with the version recorded in smf.properties
+#
+# Author Thanh Duc Do
+#
+# Last revised 30/08/2018
+
+#
+# Constants
+#
+
+readonly scriptBaseFolderPath="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+#
+# Variables
+#
+
+projectDir="$1"
+
+#
+# Check requirements
+#
+
+# Check if project dir is provided. If not: Use the scripts base directory.
+if [  -z "$1" ]; then
+	projectDir="$scriptBaseFolderPath"
+fi
+
+# Read smf.properties
+while read line; do
+	if [[ $line =~ XCODE_VERSION ]]; then
+		# Version from smf.properties
+		smf_version="${line#XCODE_VERSION=}"
+
+		# Manipulate SMF Xcode version to match Xcode's internal version representation
+		check_version="${smf_version//.}"
```

Comment:
I think it's not clear what you do for other people here. Please adjust the comment and explain that you remove the dots as you want to transfer it from eg 9.4.1 to 941 and later to 0941